api/Element.json

```diff
@@ -2907,10 +2907,10 @@
               "version_added": "43"
             },
             "safari": {
-              "version_added": null
+              "version_added": false
```

Comment:
There's a "webkitfullscreenchange" event, it should go back as far as `element.webkitRequestFullScreen()` or `element.webkitRequestFullscreen()`, whichever is older.

Reply:
Ping @vinyldarkscratch, this is blocking this PR.